packages/fluentui/react/src/components/Animation/Animation.tsx

```diff
@@ -187,7 +187,7 @@ const Animation: React.FC<AnimationProps> & {
 
     return getStyles({
       className: Animation.className,
-      displayName: Animation.displayName,
+      displayNames: [Animation.displayName],
```

Comment:
Should we have both options: displayName and displayNames? May complicate implementation, but for cases like this make sense... I'd be ok with any outcome..

Reply:
It's our internal API, so I would say that it should be more strict

Reply:
Got it, thought is `useStyles`, not I see that it is `getStyles`. Makes sense!